CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## master (unreleased)
 
+- Add support for global `before_all` tags
+```ruby
+TestProf::BeforeAll.configure do |config|
+  config.before(:begin, reset_sequences: proc(&:present?)) do
```

Comment:
Can we use RSpec API for specifying tags? I mean:

```ruby
config.before(:begin, reset_sequences: true) do
end

# or
config.before(:begin, type: :system) do
end
```

?

We need to support both presence (`true`) and equality.

UPD: I see that using RSpec utils was in the proposal; I don't think we need to be 100% compatible with RSpec, the feature should cover common cases (and common sense, which is not passing procs as Hash keys). Let's make it simpler.